File: 200_code_python/5.py
# Viết bằng PYTHON
# Tính thương hai số: Nhập hai số và in ra thương của chúng.

# Nhập hai số từ bàn phím
a = float(input("Nhap a: "))
b = float(input("Nhap b: "))

# Kiểm tra nếu b khác 0
if b != 0 :
    #Tính thương hai số
    T = a/b
    #In kết quả
    print("Thuong hai so: ", T)
else: 
    print("Khong chia duoc cho 0.")

File: 200_code_cpp/5.cpp
// Viết bằng C++
// Tính thương hai số: Nhập hai số và in ra thương của chúng.

#include <iostream>
using namespace std;

int main() {
    // Khai báo hai biến
    float a,b;

    //Nhập hai số từ bàn phím
    cout << "Nhap a: ";
    cin >> a;
    cout << "Nhap b: ";
    cin >> b;

    // Kiểm tra nếu b khác 0
    if (b != 0) {
        // Tính thương của hai số
        float T = a/b;
        // In kết quả
        cout << "Thuong cua hai so la: " << T << endl;
    } else {
        cout << "Khong the chia cho 0." << endl;
    }

    return 0;
}